rotkehlchen/api/v1/schemas.py

```diff
@@ -444,6 +450,15 @@ def validate_tx_refs(self, data: dict[str, Any], **kwargs: Any) -> None:
             tx_ref_field.deserialize_string_value(tx_ref)
             for tx_ref in data['tx_refs']
         ]
+        if (custom_indexers_order := data.get('custom_indexers_order')) is not None:
+            if not data['chain'].is_evm():
+                serialized = ', '.join(indexer.serialize() for indexer in custom_indexers_order)
+                raise ValidationError(
+                    f'Custom indexers [{serialized}] can only be used for EVM chains',
+                    field_name='custom_indexers_order',
+                )
+
+            _validate_default_indexers(custom_indexers_order)
```

Comment:
You would need to rename this, as it's now any list of indexers.
```suggestion
            _validate_indexers_list(custom_indexers_order)
```

Or ... I mean it's just running `_settings_list_is_valid` and raises a custom error if failure. Shouldn't that be another function about raising ValidationError and just use that all around by just changing the message?